topics/transcriptomics/tutorials/mirna-target-finder/tutorial.md

```diff
@@ -425,7 +425,7 @@ Repeat the filtering step on the imported DESeq2 result.
 >    - {% icon param-file %} *"Filter"*: `Differentially expressed miRNAs`
 >    - *"With following condition"*: `c3>0`
 > 4. Rename the output as `Upregulated miRNAs`
-> 5. {% tool [Sort](sort1) %} data in ascending or descending order (Galaxy Version 1.1.0) with the following parameters:
+> 5. {% tool Sort %} data in ascending or descending order (Galaxy Version 1.1.0) with the following parameters:
```

Comment:
Was that not working @pavanvidem? The tool ID looks correct: https://usegalaxy.eu/root?tool_id=sort1

Reply:
the linter complained, it wasn't on the allow-list of non-TS tools. Could you revert this back to referencing the tool and add the tool ID in the list in ./bin/check-broken-tool-links.sh ?